### PR TITLE
ROX-11349: Add rationale and guidance text according to product feedback

### PR DESCRIPTION
**Pull request by fredrb**
## Description

Adds text fields updated according to product's suggestion in [ROX-11110](https://issues.redhat.com/browse/ROX-11110). 

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] ~~Unit test and regression tests added~~
- [x] Evaluated and added CHANGELOG entry if required
- [ ] ~~Determined and documented upgrade steps~~
- [ ] ~~Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~~

### CODE REVIEW

File: migrator/migrations/m_103_to_m_104_networkpolicy_guidance/policies_before_and_after/after/deployment_has_ingress_network_policy.json [hunk 1]
```diff
@@ -0,0 +1,149 @@
+{
+  "id": "38bf79e7-48bf-4ab1-b72f-38e8ad8b4ec3",
+  "name": "Deployments should have at least one ingress Network Policy",
+  "description": "Alerts if deployments are missing an ingress Network Policy",
+  "rationale": "Pods that lack ingress Network Policies have unrestricted reachability on the network and may be exposed to attack",
```
Comment (vikin91) [inline migrator/migrations/m_103_to_m_104_networkpolicy_guidance/policies_before_and_after/after/deployment_has_ingress_network_policy.json:5]: How about using plural here? Or adding `an` before the `attack` word if we keep it singular?

```suggestion
  "rationale": "Pods that lack ingress Network Policies have unrestricted reachability on the network and may be exposed to attacks",
```

**Review by vikin91 — APPROVED** [review]
LGTM!

The comments are strictly cosmetic and are optional.

File: migrator/migrations/m_103_to_m_104_networkpolicy_guidance/policies_before_and_after/after/deployment_has_ingress_network_policy.json (hunk 1 above)
Comment (fredrb) [reply migrator/migrations/m_103_to_m_104_networkpolicy_guidance/policies_before_and_after/after/deployment_has_ingress_network_policy.json:5]: That's probably a typo yea, I copied from Chris' message in [ROX-11110](https://issues.redhat.com/browse/ROX-11110).

**Pull request merged**